concourse/pipelines/create-deployer.yml

```diff
@@ -413,6 +413,7 @@ jobs:
     serial: true
     plan:
     - get: paas-cf
+      passed: [vpc]
```

Comment:
This pins the git repo. So that the last TF runs with the same code as VPC TF. Without this, if you commit meanwhile (because you work on your branch), the last step runs with different code version and can break.